src/server/services/bezwaren/bezwaren.test.ts

```diff
@@ -10,7 +10,10 @@ import {
   fetchBezwaarDocument,
   fetchBezwaren,
   fetchBezwarenNotifications,
+  forTesting,
 } from './bezwaren';
+import { range } from '../../../universal/helpers/utils';
```

Comment:
I see that lodash is used in some places, why not here?

Reply:
This simple little util suffices.